mri/operators/fourier/non_cartesian.py

```diff
@@ -526,29 +537,26 @@ def op(self, data):
             coeff = [self._op(data[i])
                      for i in range(self.n_coils)]
         coeff = np.asarray(coeff)
+
         return coeff
 
     def _adj_op(self, coeff):
         coeff = coeff[self.sort_pos]
         stacks = np.reshape(coeff, (self.acq_num_slices, self.stack_len))
         # Receive First Fourier transformed data (per plane) in Nz x N x N
```

Comment:
It looks like based on code update, you are organizing data as `N x N x Nz`. Can you please update this comment?
Also, it would help to mention this in doc if not done already.

Reply:
Comment updated. There is already mention of the shape N x N x Nz in the function doc, I don't touch it.